bbc/simorgh
# WSTEAM1-760: Alter embed semantics

Resolves [WSTEAM1-760
](https://jira.dev.bbc.co.uk/browse/WSTEAM1-760)

Overall changes
======
Changes semantics of embeds, so that they do not appear as `<Figure />`. 

Code changes
======
- _`src/app/legacy/psammead/psammead-social-embed/src/index.jsx` changed to create a unique id for the warning label._
- `/src/app/legacy/psammead/psammead-social-embed/src/SkipLinkWrapper/index.jsx` changed to include an `aria-describedby` property, which refers to the id of the warning label.

Testing
======
1. _Visit: [https://www.test.bbc.com/pidgin/live/c7p765ynk9qt](https://www.test.bbc.com/pidgin/live/c7p765ynk9qt)._
2. _Check screenreader UX for embeds, specifically check that embeds are NOT described as Figure._

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
